app/package/publish.help

```diff
@@ -1,6 +1,12 @@
 PROJECT  - The full name of the package you wish to publish.
            Example: sample-package
```

Comment:
I think we adjust the indentation to match the new `TARGET_PLATFORM` parameter's so the display can be more consistent.